eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```diff
@@ -179,6 +194,22 @@ jobs:
       scenarios:
         - normal
 
+  # Hybrid Globalization tests
+  # ToDo: add AOT version when this is confirmed to work
+  - template: /eng/pipelines/common/templates/wasm-library-tests.yml
+    parameters:
+      platforms:
+        - browser_wasm
+        - browser_wasm_win
+      nameSuffix: _HybridGlobalization
+      extraHelixArgs: /p:HybridGlobalization=true
+      isExtraPlatformsBuild: ${{ parameters.isExtraPlatformsBuild }}
+      isWasmOnlyBuild: ${{ parameters.isWasmOnlyBuild }}
+      scenarios:
+        - normal
```

Comment:
Is running on v8 useful?

Reply:
It is not a high priority but we expect the tests to pass on each host, on V8 as well, e.g. see in https://github.com/dotnet/runtime/blob/65d38607acb332f2ccd1fbf7aed2e37f659a4d56/src/libraries/System.Runtime/tests/System.Globalization.Tests/CompareInfo/CompareInfoTests.IsPrefix.cs#L82
we detect the differences in behavior between hosts and changed the expected results accordingly.